tests/test_sandhi.py

```diff
@@ -39,11 +40,37 @@ def test_sandhi_split(sandhiobj, sandhi_reference):
     assert sandhi_reference[0] in splits
 
 def load_reference_data():
+    files = ['refs.txt',
```

Comment:
Made this change to ensure we have some tests passing to validate functionality. We can uncomment the files one by one and fix the failing test cases.

Reply:
It'd be good to have a way to extract the passing/failing tests from the UoHD files so we can add the passing ones to our regression set, and have an easy-to-work-on list of failing tests. @avinashvarna  was working on sandhi, but is currently occupied, I think.  

Something similar needs to be done on the LexicalAnalyzer tests as well. I've been off working on the Morphological Analyzer, and have neglected this.